_codux/boards/ui-kits/ui-kit-components.board.tsx

```diff
@@ -30,12 +30,31 @@ export default createBoard({
                     className={classNames(styles.productCard, styles.productCard)}
                 />
             </Variant>
-            <p className={styles.productCardInfo}>Product Card No Image</p>
+            <p className={styles.productCardInfo}>Product Card - No Image</p>
+            <Variant name="Product Card Without Image">
+                <ProductCard
+                    name="Im a product"
+                    imageUrl="https://wixmp-b7f7090100b13623109851bc.wixmp.com/layouters-starters/img_02.jpg"
+                    price={{ formatted: { price: '$15.00' } }}
+                    className={classNames(styles.productCard, styles.productCard)}
+                    outOfStock={true}
```

Comment:
should the name of the variant specify out of stock ?

Reply:
also it says no image, but I can see image url

Reply:
is the name simply wrong ?

Reply:
the title of the card is below the card -not above